sessionize/app/src/main/java/co/touchlab/sessionize/NotificationPublisher.kt

```diff
@@ -23,16 +23,18 @@ class NotificationPublisher : BroadcastReceiver() {
         val notification = intent.getParcelableExtra<Notification>(NOTIFICATION)
         val notificationId = intent.getIntExtra(NOTIFICATION_ID, 0)
 
-        with(NotificationManagerCompat.from(AndroidAppContext.app)) {
-            // notificationId is a unique int for each notification that you must define
-            this.notify(notificationId, notification)
-            Log.i(TAG,"Showing Local Notification $notificationId")
-        }
-        if(notificationId == notificationReminderId) {
-            NotificationsModel.recreateReminderNotifications()
-        }
-        if(notificationId == notificationFeedbackId){
-            NotificationsModel.recreateFeedbackNotifications()
+        notification?.let {
```

Comment:
How could notification be null? don't you only send these intents with notifications?

Reply:
I'm not sure, we are making pending intents without notifications when we cancel, but that's the only time. Either the notification or the intent is null but they shouldnt be able to be null.

Reply:
The crash says attempting to read extras from null, which makes me think intent, but the crash is on line 28 which is the notify line.